portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+/** Models a calendar event. It can be used for both creation and import flow. */
+public class CalendarEvent {
+  private String name;
+  private final String startTimeString;
```

Comment:
I think these variables should be Instant. You can perform the conversion in the constructor (that takes in String's).

Reply:
I see. Updated accordingly.